doc/classes/BaseMaterial3D.xml

```diff
@@ -603,6 +603,9 @@
 		<constant name="BLEND_MODE_MUL" value="3" enum="BlendMode">
 			The color of the object is multiplied by the background.
 		</constant>
+		<constant name="BLEND_MODE_PMALPHA" value="4" enum="BlendMode">
+			The color of the object is added to the background and the alpha channel is used to mask out the background.  This is effectively a hybrid of blend and add mode, useful for FX like fire where you want the flame to add but the smoke to mix.
```

Comment:
```suggestion
			The color of the object is added to the background and the alpha channel is used to mask out the background.  This is effectively a hybrid of blend mix and add mode, useful for FX like fire where you want the flame to add but the smoke to mix.
```

Reply:
Maybe add another sentence saying that this by default works with unshaded materials with premultiplied textures. If you write your own shader, or use a shaded material, you will need to manually premultiply.